### PR TITLE
Customer.io Cloud Extract source

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Create doc for Customer.io Cloud Extract source

### CODE REVIEW

**Review by ssbeefeater — COMMENTED** [review]
I think we should also mention that works only for customer.io accounts in US datacenters.
https://www.notion.so/rudderstacks/Customer-io-41558a56c09f458e9cf1303b3706cccd#d21bd5cb63ca4c96933bf0dff1ea9664

File: src/data/sources.yaml [hunk 1]
```diff
@@ -510,6 +510,14 @@
   category: singer-protocol
   createdAt: '2021-12-21T14:33:08.868Z'
   updatedAt: '2021-12-21T14:33:08.868Z'
+- options:
+  name: singer-chargebee
```
Comment (nidhilashkari17) [inline src/data/sources.yaml:514]: @ishachopra0401 Should this be customer io ?

Comment (ishachopra0401) [reply src/data/sources.yaml:514]: @nidhilashkari17 This does not reflect anywhere, but yeah, we should keep it consistent. I ll make the change. Thanks for pointing this out!

File: docs/sources/extract/customerio.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+title: "Customer.io"
+description: Step-by-step guide on ingesting data from Customer.io into RudderStack.
+aliases: ["Customer.io source"]
+---
+
+[Customer.io](https://www.Customer.io.com/) is a popular marketing platform for sending targeted emails and push and SMS notifications to improve customer engagement and thereby the overall conversion rate.
```
Comment (ameypv-rudder) [inline docs/sources/extract/customerio.mdx:7]:
```suggestion
[Customer.io](https://www.Customer.io.com/) is a popular marketing platform for sending targeted emails and push and SMS notifications to improve conversions and customer engagement.
```

File: docs/sources/extract/customerio.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+3. Assign a name to your source and click **Continue**.
+
+### Connection settings
+
+To set up Customer.io as a Cloud Extract source, you need to configure the following settings:
+
+<img src="../../assets/cloud-extract-sources/customerio-connection-settings.png" alt="Customer.io credentials" />
+
+- **App API Key**: Enter your Customer.io API key which can be obtained in the [Customer.io dashboard](https://fly.customer.io/login) by navigating to **Settings** > **Account Settings** > **API Credentials**.
+- **Cutoff Days**: Enter the number of days after which the updated data should be fetched.
```
Comment (ameypv-rudder) [inline docs/sources/extract/customerio.mdx:30]:
```suggestion
- **Cutoff Days**: Enter the number of days after which RudderStack fetches the updated data.
```

**Review by ameypv-rudder — APPROVED** [review]
LGTM. We should be good after incorporating the other suggestions.

**Comment by ishachopra0401** [comment]
> I think we should also mention that works only for customer.io accounts in US data-centers. https://www.notion.so/rudderstacks/Customer-io-41558a56c09f458e9cf1303b3706cccd#d21bd5cb63ca4c96933bf0dff1ea9664

Thanks for pointing this out @ssbeefeater. I have added this in the doc.

**Review by ssbeefeater — APPROVED** [review]
Looks good!

**Pull request merged**